src/main/java/kitchenpos/table/domain/NumberOdGuests.java

```diff
@@ -0,0 +1,32 @@
+package kitchenpos.table.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class NumberOdGuests {
```

Comment:
중요한 부분은 아니지만.. 오타가 있네요

```suggestion
public class NumberOfGuests {
```

Reply:
앗. 한번에 많은 양을 처리하다보니 오타난줄도 모르고 있었네요.😱 클래스명도 수정하였습니다!